src/aics-image-viewer/components/Toolbar/index.tsx

```diff
@@ -58,6 +59,7 @@ export default class Toolbar extends React.Component<ToolbarProps, ToolbarState>
       scrollBtnLeft: false,
       scrollBtnRight: false,
     };
+    this.containerRef = React.createRef();
```

Comment:
If you can believe it, I converted this to a function component at one point, then back into a class component because I thought it was more convenient! Seeing the old `createRef` makes me think I ought to come back again and switch it back now that I'm more familiar with React's new functional preference.